karabo_data/geometry2.py

```diff
@@ -205,6 +217,141 @@ def inspect(self, frontview=True):
 
         return ax
 
+    def _tile_dims(self, modno, tileno):
```

Comment:
I think this doesn't actually need the module number - at least for now, every module has the same layout of tiles with respect to how the data is stored.